### ISSUE TITLE
Ash Cap Pikachus have no image

### ISSUE THREAD

**Issue by igarvey** [felixphew/pokepaste#41]
Ash Cap Pikachus have no image. If you don't want to add each Ash Cap Pikachu, could you at least add normal Pikachu for now?

**Comment by felixphew** [comment]
Yeah, I have art files for them (which are identical to the regular Pikachu art, but they're a seperate file on Global Link so my script pulls them separately) but I don't know which ID maps to which hat. I'll have a look at some of the datamines tomorrow morning so I can get it right.

And sorry for the delay :)
